packages/terra-collapsible-menu-view/src/CollapsibleMenuViewItemGroup.jsx

```diff
@@ -80,7 +80,7 @@ class CollapsibleMenuViewItemGroup extends React.Component {
     } if (isCollapsibleMenuItem) {
       return (
         <li>
-          <List {...customProps} onChange={this.handleOnChange}>
+          <List {...customProps} role="listbox">
```

Comment:
Did we need to pass the `onChange` still?

Reply:
onChange doesn't do anything for List. I believe it was added by mistake, onChange only triggers from a selectable variant of list.